plugins/gatsby-theme-iterative-docs/gatsby-config.js

```diff
@@ -0,0 +1,57 @@
+module.exports = ({ remark }) => {
+  const plugins = []
+  if (remark) {
+    plugins.push({
+      resolve: 'gatsby-transformer-remark',
+      options: {
+        plugins: [
+          'gatsby-remark-embedder',
+          {
+            resolve: 'gatsby-remark-prismjs',
+            options: {
+              noInlineHighlight: true,
+              languageExtensions: [
+                {
+                  language: 'text',
+                  definition: {}
+                }
+              ]
+            }
+          },
+          {
+            resolve: 'gatsby-remark-smartypants',
+            options: {
+              quotes: false
+            }
+          },
+          {
+            resolve: 'gatsby-remark-embed-gist',
+            options: {
+              includeDefaultCss: true
+            }
+          },
+          'gatsby-remark-relative-images',
+          'gatsby-remark-copy-linked-files',
+          'gatsby-remark-external-links',
+          {
+            resolve: 'gatsby-remark-autolink-headers',
+            options: {
+              enableCustomId: true,
+              isIconAfterHeader: true
+            }
+          },
+          {
+            resolve: 'gatsby-remark-images',
+            options: {
+              withWebp: true
+            }
+          },
+          'gatsby-remark-responsive-iframe'
+        ]
+      }
+    })
+  }
+  return {
+    plugins
+  }
+}
```

Comment:
Reimplemented #3143

Reply:
Is there any website where we don't use remark plugins?

Reply:
iterative.ai doesn't, and likely won't, but it also wouldn't be using this theme. Also, cml.dev doesn't use the custom local remark plugins that dvc.org does, though it'll probably want to in the future.